oaknational/Oak-Web-Application
# Update MVs which filter out cycle 2 programmes

## Description
Update the curriculum MVs which filter out cycle 2 programmes from curriculum visualiser until ready for launch.

## Issue(s)

Fixes `CUR-788`

## How to test
Test that there are no functional changes to the visualiser. Also @rightsaidjames are you able to find out if we can publish a programme on staging double check it all works correctly prior to production publish?

FYI: I can setup, a test env for staging locally and proxy it over to you.
